release notes/v0.27.0.md

```diff
@@ -0,0 +1,81 @@
+- Ramping VUs: a variable number of VUs execute as many iterations as possible for a specified amount of time. This executor is equivalent to the global `stages` option.
+- Externally controlled: control and scale execution at runtime via k6's REST API or the CLI.
+
+And new executors were added to support some of the most frequently requested scenarios that were previously difficult or impossible to model:
+
+- Per VU iterations: each VU executes a fixed number of iterations (#381).
+- Constant arrival rate: a fixed number of iterations are executed in a specified period of time. This allows k6 to dynamically change the amount of VUs during a test run to achieve the specified amount of iterations per period, which is useful for a more accurate representation of RPS, for example. See #550 for details.
+- Ramping arrival rate: a variable number of iterations are executed in a specified period of time. This is similar to the ramping VUs executor, but for iterations instead.
+
+#### Execution improvements
+
+- Several executors can be configured in a single test run, both sequentially and in parallel (closes #239).
+- Each executor has a `startTime` property, which defines at what time relative to the test run start it will start executing.
+- Executors have a new `gracefulStop` property that allows for iterations to complete gracefully for some amount of time after the normal executor duration is over (closes #879); even with `gracefulStop=0`, i.e. the old behavior, metrics are emitted and processed in a much saner way (closes #1033).
+- The new `--execution-segment` and `--execution-segment-sequence` options allow easy partitioning of test runs across multiple k6 instances. Initially this applies to configuration options like number of VUs or iterations, but opens the door to test data partitioning, an often requested feature. See #997 for more details.
+- Different executors can execute different functions other than the `default` exported one. This offers more flexibility for organizing your tests, building test suites, etc.
+- Different executors can have different environment variables and tags.
+
+
+## UX
+
+- CLI: There are separate descriptions and real-time thread-safe progress bars for each individual executor.
+- A new method to stop engine execution via the REST API (#1352). Thanks @hynd!
+
+## Bugs fixed!
+- CLI: Stop `--http-debug` from exiting k6 on request dump error (#1402). Thanks @berndhartzer!
+- CLI: JSON output is now less noisy (#1469). Thanks @michiboo!
+- CLI: k6 doesn't exit when using `iterations` with `stages` (#812).
+- CLI: Mismatch in check counts in the end-of-test summary (#1033).
+- Config: Better validation of `stages` (#875).
+- JS: Rare panics in goja (#867).
+- HTTP: Fix request timeout and wrong context used for pushing metrics (#1260).
+
```

Comment:
Also, we should probably mention the websocket bugfix that @MStoykov did https://github.com/loadimpact/k6/pull/1548

Reply:
Done in 6f440313.